internal/presentation/snapshots.go

```diff
@@ -327,7 +327,12 @@ func ToSnapshotSummary(s storage.Snapshot) (tmp SnapshotSummary) {
 		Name:              s.Name,
 		Ready:             s.Ready,
 		RelatedActions:    []RelatedAction{},
-		UniqueDeviceCount: len(s.Devices),
+		UniqueDeviceCount: s.UniqueDeviceCount,
+	}
+
```

Comment:
im confused by this code block, why doesnt s.UniqueDeviceCount always work?

Reply:
FAS now stores the `Unique Device Count` in the database because the devices are stored separate and not in the Devices array.  Old snapshots did not store the `Unique Device Count`, so on those, this will be zero, but did store devices in the Devices array, so we can get the count from the length of that array.